packages/experimental/Avatar/src/JSAvatar.tsx

```diff
@@ -54,7 +60,7 @@ export const JSAvatar = compose<JSAvatarType>({
           )}
           {showRing && !transparentRing && <Slots.ring />}
           {activeAppearance === 'glow' && <Slots.glow />}
-          {!!iconSource && !!iconSource.uri && <Slots.icon source={iconSource} />}
+          <PresenceBadge styles={badgeStyles} size="small" presence={presence} isOutOfOffice={isOutOfOffice} />
```

Comment:
Should this be a slot? Otherwise users can't customize the PresenceBadge

Reply:
Yes, it should  be as a slot, but currently PresenceBadge use customize for Badge. I'm not sure if it's possible to use customize for slot. And currently if I use PresenceBadge as a slot, there's an error. That's why I want to try to use Compressible. Maybe work with styling would be easier this way